hyundai_canfd.dbc

```diff
@@ -212,7 +212,7 @@ BO_ 416 CRUISE_INFO: 32 ADRV
  SG_ ZEROS_4 : 183|4@0+ (1,0) [0|63] "" XXX
  SG_ ZEROS_6 : 119|16@0+ (1,0) [0|65535] "" XXX
  SG_ ZEROS_7 : 63|8@0+ (1,0) [0|255] "" XXX
- SG_ SET_SPEED : 102|7@0+ (1,0) [0|127] "km/h or mph" XXX
+ SG_ SET_SPEED : 102|8@0+ (0.5,0) [0|255] "km/h or mph" XXX
```

Comment:
I don't think you need to change the scaling to 0.5

Reply:
from looking at it in Cabana I think the values are else double then what they should be (the speed limit):

![image](https://user-images.githubusercontent.com/111085918/198113558-82952018-b228-4b33-bfa0-0a50532da67f.png)

vs

![image](https://user-images.githubusercontent.com/111085918/198113670-d4e70dab-fbc2-4c51-bcc9-4cb682a937f3.png)

Reply:
In the video you changed the set speed to 180km/h, where is that in the graph/cabana?

I don't think the factor should be 0.5 because if you have an 8 bit signal (2^8) then 2^8 * 0.5 is only 128 again.

Reply:
here is the chart from cabana from the video:

![image](https://user-images.githubusercontent.com/111085918/198238242-4bab294b-456f-44c1-83b8-376f593c75a0.png)

see the orange house in the background and the grey car to the left. The cart is going only till 127 as you can see. 

Thank you for explaining the math to me. I will check further until then.

Reply:
above is the original dbc file. When i do the following change its reflecting reality:

![image](https://user-images.githubusercontent.com/111085918/198239064-276ee9de-c512-4163-af96-f30c889fb509.png)

 SG_ SET_SPEED : 103|8@0+ (1,0) [0|127] "km/h or mph" XXX

No factor anymore used and I believe we have the solution. I have updated the PR. Only potential problem I can see is that bit 103 is occupied now by 2 signals:

 SG_ NEW_SIGNAL_4 : 103|1@1+ (1,0) [0|1] "" XXX

What do you suggest?

Reply:
It looks like NEW_SIGNAL_4 is actually nowhere used. I remove it in this PR. Let me know if I missed something.

Reply:
@sunnyhaibin does this look okay? I see you added that in https://github.com/commaai/opendbc/pull/673

Reply:
here is a drive with the new DBC values/changes: 2833de0636d26d55|2022-10-28--10-39-21--0 

No issues whatsoever!

Reply:
just wondering what NEW_SIGNAL_4 might have been used for, it looks like it's in [`carstate.py`](https://github.com/commaai/openpilot/blob/master/selfdrive/car/hyundai/carstate.py#L510) but not used for anything right now, but that would have to be removed too

Reply:
@incognitojam Looks good to me! I believe when the DBC for HKG CAN-FD was initially generated, bit 103 on 0x1A0 might have been accidentally labeled as its own signal. I checked the route attached and it matches with what's showing on the dashboard and openpilot's "MAX" value, which previously `CRUISE_INFO | SET_SPEED` would "reset" back to 0 and incremented when the set speed on the car's dashboard continued to increase.

![image](https://user-images.githubusercontent.com/47793918/198712977-1f013c01-65df-4c20-8625-4add39bc7232.png)